contentcuration/contentcuration/frontend/shared/views/form/ContentDefaults.vue

```diff
@@ -0,0 +1,220 @@
+        @change="emitChange"
+      />
+      <VTextField
+        v-model="aggregator"
+        outline
+        data-name="aggregator"
+        :label="$tr('aggregator')"
+        :placeholder="$tr('aggregatorPlaceholder')"
+        @change="emitChange"
+      />
+      <VTextField
+        v-model="copyrightHolder"
+        outline
+        data-name="copyrightHolder"
+        :label="$tr('copyrightHolder')"
+        :placeholder="$tr('copyrightHolderPlaceholder')"
+        @change="emitChange"
+      />
+      <VSelect
+        v-model="license"
+        outline
+        data-name="license"
+        :items="licenseOpts"
+        :label="$tr('license')"
+        @change="emitChange"
+      />
+      <VTextarea
+        v-if="isCustomLicense"
+        v-model="licenseDescription"
+        outline
+        data-name="licenseDescription"
+        :label="$tr('licenseDescription')"
+        :placeholder="$tr('licenseDescriptionPlaceholder')"
+        @change="emitChange"
+      />
+    </fieldset>
+
+    <fieldset class="py-1 mt-3">
+      <legend class="py-1 title font-weight-bold">
+        {{ thumbnailsSubTitle || $tr('thumbnailsTitle') }}
+      </legend>
+
+      <AppCheckbox
+        v-model="autoDeriveVideoThumbnail"
+        class="mt-2"
+        data-name="autoDeriveVideoThumbnail"
+        :label="$tr('videos')"
+        @change="emitChange"
+      />
+      <AppCheckbox
+        v-model="autoDeriveAudioThumbnail"
+        class="mt-2"
+        data-name="autoDeriveAudioThumbnail"
+        :label="constantStrings('audio')"
+        @change="emitChange"
+      />
+      <AppCheckbox
+        v-model="autoDeriveHtml5Thumbnail"
+        class="mt-2"
+        data-name="autoDeriveHtml5Thumbnail"
+        :label="$tr('html5')"
+        @change="emitChange"
+      />
+      <AppCheckbox
+        v-model="autoDeriveDocumentThumbnail"
+        class="mt-2"
+        data-name="autoDeriveDocumentThumbnail"
+        :label="$tr('documents')"
+        @change="emitChange"
+      />
+    </fieldset>
+  </div>
+
+</template>
+
+
+<script>
+
+  import defaultTo from 'lodash/defaultTo';
+  import { constantsTranslationMixin } from '../../../shared/mixins';
+  import AppCheckbox from './AppCheckbox';
+  import Constants from 'edit_channel/constants/index';
+  import { ContentDefaults } from 'shared/constants';
+
+  function findLicense(name, defaultValue = {}) {
+    return Constants.Licenses.find(license => license.license_name === name) || defaultValue;
+  }
+
+  export default {
+    name: 'ContentDefaults',
+    components: {
+      AppCheckbox,
+    },
+    mixins: [constantsTranslationMixin],
+    model: {
+      prop: 'contentDefaults',
+      event: 'change',
+    },
+    props: {
+      contentDefaults: {
+        type: Object,
+      },
+      defaultsTitle: {
+        type: String,
+      },
+      defaultsSubTitle: {
+        type: String,
+      },
+      thumbnailsSubTitle: {
+        type: String,
+      },
+    },
+    data() {
```

Comment:
I was wondering if it would make sense to save default values (well to be precise I mean defaults default values :) for fields in a config file so we can find/update them easily. It could be also used for comparison [here](https://github.com/learningequality/studio/pull/1746/files#diff-8ba08320f6560ac74479458a89786339R106) instead of defining them again in specs.

Reply:
Yeah, that couldn't hurt. I'll try it out.